### ISSUE TITLE
Question about model

### ISSUE THREAD

**Issue by Jiawei0o0** [Wangyixinxin/ACN#5]
https://github.com/Wangyixinxin/ACN/blob/ae931b1a2471a8f84346ef63c6515cee24681643/models/build.py#L10
Hi, thanks for your great work. I am curious about why your output channel is four in all models. Shouldn't it be set to three? Or I miss something here?

**Comment by Wangyixinxin** [comment]
Hi, you can refer to here: https://github.com/Wangyixinxin/ACN/blob/ae931b1a2471a8f84346ef63c6515cee24681643/data.py#L36-L39

**Comment by VoooWei** [comment]
Hi, Sorry to bother you. I try to build a model with your reference code [https://github.com/doublechenching/brats_segmentation-pytorch/blob/master/data.py#L38](Reproduction ) and I noticed that you two make difference on the code "data.py".  Labels are defined in different ways.  So, the output channel is 3 in the reference code and yours is 4. What's the difference between them? Thaks for you reply!

**Comment by Wangyixinxin** [comment]
Hi, they output the area of three-subregions(i.e. the whole tumor, tumor core & enhancing tumor) and we output the area of three labels (ed, net, et). The final goal is the same.

**Comment by VoooWei** [comment]
Hi, sorry to bother you. I have a question about the KL loss. I used your code directly but got negative values on the KL loss.  I found that this is caused by the fact that you did not take the softmax operation on the input of F.kl_div function.  Am I missing something here?

**Comment by Wangyixinxin** [comment]
Hi thanks for pointing out the missing kl loss! The default consistency loss in these codes is mse loss. I just updated the consistency loss part for [kl loss ](https://github.com/Wangyixinxin/ACN/blob/main/losses.py#L139-L142). Tell me if you have any other question. :)

**Comment by Jiawei0o0** [comment]
Hi, could you tell me what's your baseline on T1ce modality?  I can't find it in the paper.

**Comment by Wangyixinxin** [comment]
Hi, please refer to Table 1 and Table 2. The baselines are HeMIS, HVED and KD-net. T1ce modality presents if denoted by •
![image](https://user-images.githubusercontent.com/54788216/147427282-c6e47714-890f-4bc3-9675-e2c48cd5661c.png)
![image](https://user-images.githubusercontent.com/54788216/147427406-d909cca6-c26d-49a3-aa1b-cd7958f4792e.png)
